robots-ev3-module/src/main/java/orwell/tank/communication/UdpProxyFinder.java

```diff
@@ -7,21 +7,23 @@
 import utils.BroadcastAddress;
 
 import java.io.IOException;
+import java.io.InputStream;
 import java.net.*;
+import java.util.Arrays;
 import java.util.Enumeration;
+import java.util.Scanner;
 
 public class UdpProxyFinder {
-    private final static Logger logback = LoggerFactory.getLogger(UdpProxyFinder.class);
-    private final int RECEIVER_BUFFER_SIZE = 512;
+    private static final Logger logback = LoggerFactory.getLogger(UdpProxyFinder.class);
     private final int broadcastPort;
     private final DatagramSocket datagramSocket;
     private final UdpBroadcastDataDecoder udpBroadcastDataDecoder;
-    private int attemptsPerformed = 0;
+    private final SimpleEscapeKeyListener simpleEscapeKeyListener;
     private String pushAddress;
     private String pullAddress;
-    private SimpleEscapeKeyListener simpleEscapeKeyListener;
+    private int attemptsPerformed;
```

Comment:
This should be initialized, probably to 0. The value is only incremented and logged though.

Reply:
Same. Fixed!